articles/azure-monitor/toc.yml

```diff
@@ -404,6 +404,10 @@
         items:
         - name: Node.js
           href: app/nodejs.md
+      - name: Python apps
```

Comment:
To move this to the quickstart section we would have to make a few small tweaks to this just so it would have the same structure as the other App Insights Quickstarts